far/editor.cpp

```diff
@@ -3405,7 +3398,7 @@ bool Editor::Search(bool Next)
 			catch (regex_exception const& e)
 			{
 				ReCompileErrorMessage(e, LastSearchDlgParams.SearchStr);
-				return false; //BUGBUG
+				return; // Broken regex could not and was not found. Do as if the search string was not found. Do NOT restore IsReplaceAll.
```

Comment:
This sounds a bit weird:
- "could not _be found_" perhaps?
- "and was not found" - but we didn't even try, did we?

Reply:
Yeah! Fixed. Thanks.